include/circt/Dialect/FIRRTL/OpStatements.td

```diff
@@ -125,6 +125,39 @@ def StopOp : FIRRTLOp<"stop", []> {
   }];
 }
 
+def AssertOp : FIRRTLOp<"assert", []> {
+  let summary = "Assert Verification Statement";
+
+  let arguments = (ins ClockType:$clock, UInt1Type:$predicate, UInt1Type:$enable, StrAttr:$message);
```

Comment:
Please wrap to 80 columns here and below.

Reply:
Ack... I should just setup `clang-format` as a save hook...

I ran `clang-format` over the modified region in a subsequent commit.

Reply:
Note that clang-format doesn't work on .td files... :-/

Reply:
Missed this comment and learned this from the regression tests. 😬